docs/source/operations/corda_networkyaml.md

```diff
@@ -19,13 +22,17 @@ The configurations are grouped in the following sections for better understandin
 
 * organizations
 
-`type` defines the platform choice like corda/fabric.
+Here is the snap shot of the same from the sample network.yaml
+![](./../_static/NetworkYamlCorda1.png)
```

Comment:
Please so not use images for this sections because if there is a change, we will have to take screenshots again. Use a code block like below:
```
env:
  proxy:
  ports: 

```